src/test/java/seedu/realodex/logic/parser/ArgumentTokenizerTest.java

```diff
@@ -138,13 +138,13 @@ public void tokenize_multipleArgumentsJoined() {
 
     @Test
     public void equalsMethod() {
-        Prefix aaa = new Prefix("aaa");
+        Prefix aaa = new Prefix("aaa", "aaa");
 
         assertEquals(aaa, aaa);
-        assertEquals(aaa, new Prefix("aaa"));
+        assertEquals(aaa, new Prefix("aaa", "aaa"));
 
         assertNotEquals(aaa, "aaa");
-        assertNotEquals(aaa, new Prefix("aab"));
+        assertNotEquals(aaa, new Prefix("aab", "aab"));
```

Comment:
i might have missed it but i don't see any tests for checking that prefix is now case insensitive?

Reply:
Nope no tests yet